update-netty-substitutions.sh

```diff
@@ -37,48 +57,48 @@ else
     echo "The Netty target version is: $NETTY_VERSION"
 fi
 
-# Creating "nettyhandling" and "grpchandling" directories to keep pulled files separate from our extension's
-echo "$counter - Creating nettyhandling grpchandling directories"
+# Step 2: Create directory structure for the modified files
+echo "$counter - Creating netty grpc directories"
 ((counter++))
 mkdir -p "$DEST_NETTY_DEPLOYMENT"
 mkdir -p "$DEST_NETTY_RUNTIME"
 mkdir -p "$DEST_GRPC_DEPLOYMENT"
 mkdir -p "$DEST_GRPC_RUNTIME"
 
-# Copy files from runtime and deployment directories and overwrite existing
+# Step 3: Copy files from Quarkus to our extension
 echo "$counter - Copying files"
 ((counter++))
 cp -r "$SRC_NETTY_RUNTIME"/* "$DEST_NETTY_RUNTIME"
 cp -r "$SRC_NETTY_DEPLOYMENT"/* "$DEST_NETTY_DEPLOYMENT"
-
 cp -r "$SRC_GRPC_RUNTIME"/* "$DEST_GRPC_RUNTIME"
 cp -r "$SRC_GRPC_DEPLOYMENT"/* "$DEST_GRPC_DEPLOYMENT"
 
-# Prepend "io.grpc.netty.shaded." to all "io.netty" occurrences in the copied files
+# Step 4: Update namespace references
+# Replace regular Netty references with shaded Netty references
 echo "$counter - Replacing shaded netty namespace"
 ((counter++))
 find "$DEST_NETTY_RUNTIME" "$DEST_NETTY_DEPLOYMENT" -type f -name "*.java" -exec sed -i '' 's/io\.netty/io.grpc.netty.shaded.io.netty/g' "{}" +
 find "$DEST_GRPC_RUNTIME" "$DEST_GRPC_DEPLOYMENT" -type f -name "*.java" -exec sed -i '' 's/io\.grpc\.netty/io.grpc.netty.shaded.io.grpc.netty/g' "{}" +
 find "$DEST_GRPC_RUNTIME" "$DEST_GRPC_DEPLOYMENT" -type f -name "*.java" -exec sed -i '' 's/io\.netty/io.grpc.netty.shaded.io.netty/g' "{}" +
 
-
-# Fix the imports and packages
+# Step 5: Update package declarations and imports
+# Modify package names to match our extension's structure
 echo "$counter - Fixing imports and packages"
 ((counter++))
 find "$DEST_NETTY_RUNTIME" "$DEST_NETTY_DEPLOYMENT" -type f -name "*.java" -exec sed -i '' \
-    -e 's/io\.quarkus\.netty\.deployment/io.quarkiverse.temporal.deployment.graal.nettyhandling/g' \
-    -e 's/io\.quarkus\.netty/io.quarkiverse.temporal.graal.nettyhandling/g' \
-    -e 's/io\.quarkus\.netty\.runtime/io.quarkiverse.temporal.graal.nettyhandling.runtime/g' \
-    -e 's/io\.quarkus\.netty\.runtime\.virtual/io.quarkiverse.temporal.graal.nettyhandling.runtime.virtual/g' \
-    -e 's/io\.quarkus\.netty\.runtime\.graal/io.quarkiverse.temporal.graal.nettyhandling.runtime.graal/g' \
+    -e 's/io\.quarkus\.netty\.deployment/io.quarkiverse.temporal.deployment.graal.netty/g' \
+    -e 's/io\.quarkus\.netty/io.quarkiverse.temporal.graal.netty/g' \
+    -e 's/io\.quarkus\.netty\.runtime/io.quarkiverse.temporal.graal.netty/g' \
```

Comment:
If removing the runtime from the package name the directories won't match the package name, so we would have to move the packages from the runtime folder, as after git clone we just do a `cp -r` so it was keeping the directory structure that comes from Quarkus, I wouldn't rename the package removing runtime to make it match as much as possible the source of truth